app/views/posts/index.html.erb

```diff
@@ -17,7 +17,7 @@
         <p> Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %> </p>
       </div>
       <div class = "border rounded border-black flex flex-col p-2 gap-3">
-        <% post.comments.each do | comment| %>
+        <% post.recent_comments do | comment| %>
```

Comment:
- [ ] Great work so far but it would be nice if you put `.each` after recent_comments because you are iterating here and i cannot see any comments because .each is not there 😨 Kindly put it to make your app work as expected 👍